pkg/retrieval/pb/doc.go

```diff
@@ -0,0 +1,6 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//go:generate sh -c "protoc -I . -I \"$(go list -f '{{ .Dir }}' -m github.com/gogo/protobuf)/protobuf\" --gogofaster_out=. retrieval.proto"
```

Comment:
This line should be removed separated from the package declaration by one line as it is not the package description.